content/en/methods/timelines/streaming.md

```diff
@@ -72,6 +72,13 @@ Possible `stream` values:
 * `list`
 * `direct`
 
+WebSocket events have this structure:
+
+```text
```

Comment:
I am not sure if this is accurately formatted since it looks more like the EventSource version. Maybe it should be a JSON object?